frontend/www/js/omegaup/components/contest/Editv2.vue

```diff
@@ -10,7 +10,7 @@
         <small>
           &ndash;
           <a v-bind:href="`/arena/${details.alias}/`">
-            {{ T.contestDetailsGoToContest }}</a
+            {{ T.contestDetailsoToContest }}</a
```

Comment:
```suggestion
            {{ T.contestDetailsGoToContest }}</a
```

ya tengo que terminar ese linter para advertir de usos de cadenas de traducción inexistentes.

Reply:
hurra, ya se despliega acá el error :D